test/grouping.jl

```diff
@@ -1091,4 +1106,21 @@ end
     @test eltypes(DataFrame(gd)) == [Union{Missing, Symbol}, Int]
 end
 
+@testset "groupindices and groupvars" begin
+    df = DataFrame(A = [missing, :A, :B, :A, :B, missing], B = 1:6)
+    gd = groupby_checked(df, :A)
+    @test groupindices(gd) == [1, 2, 3, 2, 3, 1]
```

Comment:
Also test type?

Reply:
I have added `@inferred` test, which should pass with the current implementation (and fails with a comprehension).